Karma/Migrations/20211114211354_chat-fix.cs

```diff
@@ -0,0 +1,86 @@
+﻿using System;
```

Comment:
I also added a migration for these new changes (in messaging). Currently it's in the master, so maybe this migration could be deleted?